packages/plugin-segment/lib/index.ts

```diff
@@ -1,15 +1,26 @@
 /* eslint-disable no-unused-vars, class-methods-use-this, import/no-unresolved */
 import {
-  Options, Event, Properties, Plugin,
+  Event, EventMetadata, EventOptions, Properties, Plugin,
 } from '@itly/sdk';
 
 export type SegmentOptions = {};
 
+/**
+ * Segment specific metadata
+ * https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#track
+ */
+export interface SegmentMetadata {
+  options?: {
```

Comment:
I moved `integrations` onto an `options` object to better match the [Segment docs](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#track).  I did the same for node to keep the "track fields" separate from the callback.

Reply:
I've add `options` to the Snowplow metadata for consistency